gpu_ocean/demos/DAPaper/scripts/run_model.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+
+"""
+This software is part of GPU Ocean. 
+
+Copyright (C) 2018 SINTEF Digital
+Copyright (C) 2018 Norwegian Meteorological Institute
+
+This python program runs a short program while reporting execution
+metrics, and is intended to be used for meassuring computational 
+performance and throughput ofthe CDKLM scheme with model errors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
```

Comment:
Add a comment on the purpose of this script.

Reply:
Already at lines 9-11